scripts/reindex_feedme_text_chunks.py

```
@@ -0,0 +1,235 @@
+            logger.info("conversation=%s chunks=0", conversation_id)
+            continue
+
+        chunk_texts: list[str] = [str(row.get("content") or "") for row in rows]
+        logger.info("conversation=%s chunks=%s dry_run=%s", conversation_id, len(rows), dry_run)
+
+        for row_idx, row in enumerate(rows):
+            chunk_id = row.get("id")
+            try:
+                chunk_index = int(row.get("chunk_index"))
+            except Exception:
+                continue
```

Ruff (0.14.10)

`try`-`except`-`continue` detected, consider logging the exception

(S112)